layers/meta-resin-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend

```diff
@@ -9,8 +9,8 @@ LINUX_VERSION = "4.9.59"
 SRCREV = "e7976b2affa8c05bd299ead0ad9c36ddaf135b9d"
 
 # Set console accordingly to build type
-DEBUG_CMDLINE = "dwc_otg.lpm_enable=0 console=tty1 console=serial0,115200 root=/dev/mmcblk0p2 rootfstype=ext4 rootwait"
-PRODUCTION_CMDLINE = "dwc_otg.lpm_enable=0 console=null root=/dev/mmcblk0p2 rootfstype=ext4 rootwait vt.global_cursor_default=0"
+DEBUG_CMDLINE = "dwc_otg.lpm_enable=0 console=tty1 console=serial0,115200 root=LABEL=resin-rootA rootdelay=5 fsck.repair=yes rootfstype=ext4 rootwait"
+PRODUCTION_CMDLINE = "dwc_otg.lpm_enable=0 console=null root=LABEL=resin-rootA rootdelay=5 fsck.repair=yes rootfstype=ext4 rootwait vt.global_cursor_default=0"
```

Comment:
Can you leave out of this PR the addition of "fsck.repair=yes" and "rootdelay=5" ?

Reply:
Please change the commit log to something like:

> linux-raspberrypi_%.bbappend: Use filesystem labels for the root cmdline kernel parameter

Followed by a detailed description on why this change is necessary

Reply:
Also, please add a CHANGELOG.md entry with a short line for this change, like in: 0ab6e8d

Reply:
Please redo this commit.
You need to remove the
> rootdelay=5 fsck.repair=yes

additions from this commit. After you do that, you can drop the other commit: https://github.com/resin-os/resin-raspberrypi/pull/165/commits/63b71ef42284670d90ceb3ed4e64a0d967eeb1c6

To do this, in your branch you should use "git rebase".